stupidns/stupidns.go

```
@@ -0,0 +1,84 @@
+		m.Answer[i] = rr
+	}
+
+	m.Response = true
+	m.Opcode = 0
+
+	s.responseQueue = append(s.responseQueue, m)
+}
+
+func (s *StupiDNS) Shutdown() {
+	s.responseQueue = nil
+	s.srv.Shutdown()
```

golangci-lint (1.62.2)

Error return value of `s.srv.Shutdown` is not checked

(errcheck)